tests/python_client/milvus_client/test_milvus_client_insert.py

```diff
@@ -1342,5 +1344,1037 @@ def test_milvus_client_insert_after_json_path_index(self, enable_dynamic_field,
+        self.release_collection(client, collection_name)
+
+        # Step 5: partial update data
+        new_row = cf.gen_row_data_by_schema(nb=default_nb, schema=schema, 
+                                            desired_field_names=[default_primary_key_field_name, default_string_field_name])
+        error = {ct.err_code: 101, 
+                 ct.err_msg: f"failed to query: collection not loaded"}
+        self.upsert(client, collection_name, new_row, partial_update=True,
+                    check_task=CheckTasks.err_res, check_items=error)
+        
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="Partial update not ready")
+    def test_milvus_client_partial_update_same_pk_after_delete(self):
+        """
+        target: test PU will fail when provided same pk and partial field
+        method:
+            1. Create a collection with dynamic field
+            2. Insert rows
+            3. delete the rows
+            4. upsert the rows with same pk and partial field
+        expected: step 4 should fail
+        """
+        # Step 1: create collection
+        client = self._client()
+        schema = self.create_schema(client, enable_dynamic_field=False)[0]
+        schema.add_field(default_primary_key_field_name, DataType.INT64, is_primary=True, auto_id=False)
+        schema.add_field(default_vector_field_name, DataType.FLOAT_VECTOR, dim=default_dim)
+        schema.add_field(default_int32_field_name, DataType.INT32)
+        index_params = self.prepare_index_params(client)[0]
+        index_params.add_index(default_primary_key_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_vector_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_int32_field_name, index_type="AUTOINDEX")
+        collection_name = cf.gen_collection_name_by_testcase_name(module_index=1)
+        self.create_collection(client, collection_name, default_dim, schema=schema, 
+                               consistency_level="Strong", index_params=index_params)
+        
+        # Step 2: insert rows
+        rows = cf.gen_row_data_by_schema(nb=default_nb, schema=schema)
+        self.upsert(client, collection_name, rows, partial_update=True)
+        
+        # Step 3: delete the rows
+        ids = [row[default_primary_key_field_name] for row in rows]
+        self.delete(client, collection_name, ids=ids)
+        
+        # Step 4: upsert the rows with same pk and partial field
+        new_rows = cf.gen_row_data_by_schema(nb=default_nb, schema=schema, 
+                                            desired_field_names=[default_primary_key_field_name, default_vector_field_name])
+        error = {ct.err_code: 1100, 
+                 ct.err_msg: f"fieldSchema({default_int32_field_name}) has no corresponding fieldData pass in: invalid parameter"}
+        self.upsert(client, collection_name, new_rows, partial_update=True,
```

Comment:
i don't think this error is related to delete request. what we need to verify is that the data persistency after delete and partial update